src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -1,7 +1,42 @@
+@import "../../patternfly-utilities";
+
+$pf-l-gallery__item--Width: 250px !default;
```

Comment:
I know where this is coming from but this variables is breaking the system. There is no width property declared.
Also please make a comment on the 250px magic number. the fact that's not refereeing to a global variable is breaking it again

Reply:
I don't really like having this here at all. In theory, I should be able to put `auto` in the minmax, but it doesn't work without a concrete size - the gallery items take the full width instead. This also doesn't really give you a way to use the gallery multiple times with different item widths. Any ideas?